ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -14,3 +14,4 @@ webserver_caching_port_container: 8080
 registry_creation: false
 url_passed: false
 httpd_cache_files: "{{ provision_cache_store }}httpd/"
+bootmode: UEFI
```

Comment:
I realized we don't need this at all. There is no check for it.

Reply:
Wouldn't we need because if bootmode is not set in the inventory (it's commented out by default), our templating of install-config.yml will fail. If we want to remove this default variable, maybe then we need to add an additional check in the install-config.j2 to check if bootmode is defined 
```
{% if bootmode is defined and bootmode == 'legacy' %}
        bootMode: legacy
{% endif %}

Reply:
@smalleni okay i will add a `if defined` check and remove that default variable.